.travis.yml

```diff
@@ -1,407 +0,0 @@
-        - composer show
-
-    - name: "PHP 7.2, Agnostic, Symfony ~4"
-      php: 7.2
-      env: DB=agnostic
-      install:
-        - rm -f ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/xdebug.ini
-        - COMPOSER=./tests/composer/composer-symfony4-min.json composer install
-        - composer show
-    - name: "PHP 7.2, Agnostic, Symfony ^4"
-      php: 7.2
-      env: DB=agnostic
-      install:
-        - rm -f ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/xdebug.ini
-        - COMPOSER=./tests/composer/composer-symfony4-max.json composer install
-        - composer show
-
-    - name: "PHP 7.2, Agnostic, Symfony ~5"
-      php: 7.2
-      env: DB=agnostic
-      install:
-        - rm -f ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/xdebug.ini
-        - COMPOSER=./tests/composer/composer-symfony5-min.json composer install
-        - composer show
-    - name: "PHP 7.2, Agnostic, Symfony ^5"
-      php: 7.2
-      env: DB=agnostic
-      install:
-        - rm -f ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/xdebug.ini
-        - COMPOSER=./tests/composer/composer-symfony5-max.json composer install
-        - composer show
-
-    - name: "PHP 7.2, MySql, Symfony ~3"
-      php: 7.2
-      env: DB=mysql
-      install:
-        - rm -f ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/xdebug.ini
-        - COMPOSER=./tests/composer/composer-symfony3-min.json composer install
-        - composer show
-    - name: "PHP 7.2, MySql, Symfony ^3"
-      php: 7.2
-      env: DB=mysql
-      install:
-        - rm -f ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/xdebug.ini
-        - COMPOSER=./tests/composer/composer-symfony3-max.json composer install
-        - composer show
-
-    - name: "PHP 7.2, MySql, Symfony ~4"
-      php: 7.2
-      env: DB=mysql
-      install:
-        - rm -f ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/xdebug.ini
-        - COMPOSER=./tests/composer/composer-symfony4-min.json composer install
-        - composer show
-    - name: "PHP 7.2, MySql, Symfony ^4"
-      php: 7.2
-      env: DB=mysql
-      install:
-        - rm -f ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/xdebug.ini
-        - COMPOSER=./tests/composer/composer-symfony4-max.json composer install
-        - composer show
-
-    - name: "PHP 7.2, MySql, Symfony ~5"
-      php: 7.2
-      env: DB=mysql
-      install:
-        - rm -f ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/xdebug.ini
-        - COMPOSER=./tests/composer/composer-symfony5-min.json composer install
-        - composer show
-    - name: "PHP 7.2, MySql, Symfony ^5"
-      php: 7.2
-      env: DB=mysql
-      install:
-        - rm -f ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/xdebug.ini
-        - COMPOSER=./tests/composer/composer-symfony5-max.json composer install
-        - composer show
-
-  fast_finish: true
-
-cache:
-  directories:
-    - $HOME/.composer/cache
-
-notifications:
```

Comment:
What about those notifications, should we also add them to GA?

Reply:
probably not for now.